src/main/java/com/facebook/openwifirrm/ucentral/UCentralClient.java

```diff
@@ -591,6 +593,78 @@ public SerialNumberList getProvInventoryForRRM() {
 		}
 	}
 
+	/**
+	 * Retrieve the RRM config and schedule for a specific AP
+	 *
+	 * @param serial the serial number of the AP
+	 *
+	 * @return {@link RRMDetails}, containing information about the RRM
+	 *   schedule and parameters
+	 */
+	public RRMDetails getProvInventoryRrmDetails(String serial) {
+		Map<String, Object> parameters = new HashMap<>();
+		parameters.put("rrmSettings", true);
+		HttpResponse<String> response =
+			httpGet(String.format("inventory/%s", serial), OWPROV_SERVICE, parameters);
+		if (!response.isSuccess()) {
+			logger.error("Error: {}", response.getBody());
+			return null;
+		}
+
+		try {
+			logger.debug("SERIAL {} BODY: {}", serial, response.getBody());
+			return gson.fromJson(response.getBody(), RRMDetails.class);
+		} catch (JsonSyntaxException e) {
+			String errMsg = String.format(
+				"Failed to deserialize to RRMDetails: %s",
+				response.getBody()
+			);
+			logger.error(errMsg, e);
+			return null;
+		}
+	}
+
+	/** Retrieve resolved inventory */
+	public InventoryTagList getProvInventoryGetConfigApplyResult(
+		String serial
+	) {
+		Map<String, Object> parameters = new HashMap<>();
+		parameters.put("resolveConfig", true);
+		HttpResponse<String> response =
+			httpGet(String.format("inventory/%s", serial), OWPROV_SERVICE, parameters);
+		if (!response.isSuccess()) {
+			logger.error("Error: {}", response.getBody());
+			return null;
+		}
+		try {
+			InventoryConfigApplyResult result = gson
+				.fromJson(response.getBody(), InventoryConfigApplyResult.class);
+			if (result.warnings != null) {
```

Comment:
Maybe also check if the `warnings` or the `errors` are empty? I see them are defined as list in `InventoryConfigApplyResult` class.